admin-docs/metadata-extraction.md

```diff
@@ -0,0 +1,101 @@
+
+The effectiveness of this process is measured by accuracy, which refers to how often the models correctly identify and extract the right information. While the models strive for high accuracy, their performance can vary based on the quality of the documents and how closely they resemble the data the models were trained on. Over time, the models can be improved with more training data, enabling them to handle a broader range of document types and complex layouts.
+
+### Limitations
+
+While the machine learning models used for metadata extraction are powerful, they do have limitations. The accuracy of extraction depends on how similar your documents are to those the models were trained on. If your documents have unusual formats, inconsistent language, or unique content, the system may require human intervention to ensure correct details are extracted. Additionally, complex or heavily unstructured documents, like lengthy legal narratives, may require more manual review after extraction. Therefore, while this tool is effective, it’s best used alongside human validation, especially for sensitive or detailed work.
+
+### Prerequisites
+
+Before using the metadata extraction tool, there are a few prerequisites to consider. These include access to the tool, the type of data it works with, and your familiarity with the document content you want to extract.
+
+- **Access**: This tool is in the beta phase and isn’t activated automatically for all Uwazi instances. Contact us if you’d like to access it during this phase. Note that when activated, the tool is only accessible to users with the roles of admin or editor.
+
+- **Available data**: The tool currently works with PDF documents and can extract information to specific template properties. Supported properties include text, numeric, date, single select, multiple select, and relationship. Your instance should have appropriate templates with documents available for metadata extraction.
+
+- **Knowledge of document content**: It’s important to have a clear understanding of the kind of information you want to extract, such as specific properties or categories relevant to your documents. This preparation will help ensure a smooth and accurate extraction process.
+
+## How to use metadata extraction effectively
+
+This step-by-step guide will walk you through the process of extracting metadata from your documents. First, initiate the metadata extraction by selecting the relevant properties from the available templates. The tool will then generate and present the extracted metadata as suggestions. Finally, review the suggestions, accepting those that are correct and refining any that need adjustments. Following these steps will help ensure that the extracted metadata is both accurate and relevant.
```

Comment:
`First, initiate the metadata extraction by selecting the relevant properties from the available templates. The tool will then generate and present the extracted metadata as suggestions. Finally, review the suggestions, accepting those that are correct and refining any that need adjustments. Following these steps will help ensure that the extracted metadata is both accurate and relevant`.

When reading this paragraph, it feels to me that we are jumping too quickly from First to Finally. I would consider using another linking word in the sentence starting with Finally or linking the sentences by using another clause.

Reply:
Resolved with [3c67ca6](https://github.com/huridocs/uwazi-documentation/pull/49/commits/3c67ca67ffee57322777d3a357d5fa4194de4551)